### DISCUSSION TITLE
Version 3.0

### DISCUSSION THREAD

**Discussion by wrongkindofdoctor** [NOAA-GFDL/MDTF-diagnostics#433] [General]
This thread is for suggestions for features, improvements, and general discussion about version 3.0 of the MDTF-diagnostics package. Notable changes in version 3.0 include:

* upgrading the required Python version from 3.8 to 3.10 to accommodate version requirements in dependencies. Users should update their conda environments after syncing their main, and POD development branches if they are working on a POD, on their remote and local forks with NOAA-GFDL repository. Users who are just running the package using a cloned copy of the NOAA-GFDL repo, or their own forks synced with the NOAA-GFDL repo,  can fetch and pull the changes into the main branch, and check out the v3.0 tag to a new branch `git checkout v3.0 -b v3.0` . Note that just running `git checkout tag v3.0` puts you in a detached head state, which won't prevent you from running the package, but will not allow you to track changes you make to you make to config files or any other modifications. 
* Eliminating the convective_transition_diag and precip_buoy_diag environment files, and adding any necessary packages to the python3 base environment file for these PODs.
* Adding support for multirun PODs that analyze multiple model and/or observational datasets, and an example_multicase POD demonstrating the configuration for mutirun mode. Multirun mode is a beta feature, and will be debugged/extended as users test the example_multicase POD with new datasets, and new multirun PODs are added to the repository.

**Comment by bitterbark** [comment]
I tested this on NCAR machine, and it works for the 4 PODs active in the distributed default_tests.yaml.

If any users of NCAR's cheyenne/caspar are interested in sharing the environments, I plan to keep versions for shared use. You can access this one at: /glade/work/bundy/miniconda2/envs.MDTFv3.0
